packages/ui/src/council/hooks/useElectedCouncil.ts

```diff
@@ -0,0 +1,8 @@
+import { useGetElectedCouncilsQuery } from '@/council/queries'
+import { asCouncil } from '@/council/types'
+
+export const useElectedCouncil = () => {
+  const { loading, data } = useGetElectedCouncilsQuery({ variables: { where: { deletedAt_all: false } } })
```

Comment:
@jodator my interpretation is that the `ElectedCouncil` gets "deleted" once the council is replaced, is that correct?

Reply:
I think that this comes from Hydra and is automatically added to all `where` variables.

Maybe `endedAtBlock` (when null) is better. ATM I don't see other field that could indicate currently elected council.

Reply:
An alternative way is to query the API:

```
api.query.council.councilMembers()
```

Reply:
Right! I always forget that passing `null` is an option